src/plugins/intel_gpu/CMakeLists.txt

```diff
@@ -44,6 +47,7 @@ include(thirdparty/cmake/rapidjson.cmake)
 if(CMAKE_COMPILER_IS_GNUCXX)
     ov_add_compiler_flags(-Werror)
 endif()
+ov_add_compiler_flags(-Rno-debug-disables-optimization)
```

Comment:
is it required? looks like it works without this flag

Reply:
It's not. IIRC, I added it to avoid huge amount of warnings in debug build